guides/private-app/src/api.ts

```diff
@@ -0,0 +1,28 @@
+// Execute request to CARTO SQL API
+interface ExecuteQueryProps {
```

Comment:
Is it not better to use plain JavaScript rather than TypeScript? For those not using TypeScript it is additional friction

Reply:
I think there is value is having pure JS, it allows the developer to just copy-paste the code into jsfiddle or similar. Do we have any idea of what our customers mostly use?

Reply:
I prefer to use TypeScript because the user of this is a Developer who wants to build a large application and Typescript is more used today for these use cases